### PR TITLE
feat(UIExplorer): Minimal UIExplorer App

### LINKED ISSUE #210: Frame UI Explorer App for Windows

### PR DESCRIPTION
**Pull request by rozele**
Get a stripped down version of the UIExplorer app working. This minimal version excludes the status bar, drawer/navigator tab bar, and removes most of the demos except for simple ones.

Fixes #210

### CODE REVIEW

File: ReactWindows/ReactNative/DevSupport/DevSupportManager.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ class DevSupportManager : IDevSupportManager
 
         private bool _isDevSupportEnabled = true;
         private bool _isShakeDetectorRegistered;
-        private bool _isUsingJsProxy;
+        private bool _isUsingJsProxy = true;
```
Comment (rozele) [inline ReactWindows/ReactNative/DevSupport/DevSupportManager.cs]: Revert this.

File: ReactWindows/Playground/AppReactPage.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ public override string MainComponentName
         {
             get
             {
-                return "ReactRoot";
+                return "UIExplorerApp";
```
Comment (rozele) [inline ReactWindows/Playground/AppReactPage.cs]: Revert this.

File: ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs [hunk 3]
```diff
@@ -1,4 +1,6 @@
-﻿using Newtonsoft.Json.Linq;
+﻿#define CLEAR_STORAGE
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs]: Revert.

File: ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs [hunk 4]
```diff
@@ -1,4 +1,6 @@
...
+
+using Newtonsoft.Json.Linq;
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs]: Needed?

File: ReactWindows/ReactNative/ReactPage.cs [hunk 5]
```diff
@@ -106,6 +107,11 @@ public void OnSuspend()
         public void OnResume()
```
Comment (rozele) [inline ReactWindows/ReactNative/ReactPage.cs]: Perhaps the back button handler should be registered in OnCreate?

**Comment by rozele** [comment]
@mpodwysocki any comments?

**Comment by mpodwysocki** [comment]
@rozele sorry, getting to it now

File: ReactWindows/js/CustomComponents/NavigatorBreadcrumbNavigationBarStyles.windows.js [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+ * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THE SOFTWARE, EVEN IF
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @providesModule NavigatorBreadcrumbNavigationBarStyles
+ */
+'use strict';
+
+var Dimensions = require('Dimensions');
+var NavigatorNavigationBarStylesIOS = require('NavigatorNavigationBarStylesIOS');
```
Comment (mpodwysocki) [inline ReactWindows/js/CustomComponents/NavigatorBreadcrumbNavigationBarStyles.windows.js]: Should we keep the iOS style or do our own for UWP?

**Pull request merged**